blog/docs/articles/improved-ha-configuration.md

```diff
@@ -0,0 +1,82 @@
+# Improving HA configuration for Knative workloads
+
+**Author: Matthias Weßendorf, Senior Principal Software Engineer @ Red Hat**
+
+**Date: 2023-06-13**
+
+_In this blog post you will learn how to use the Knative Operator to maintain a fine-grain configuration for high availablitly of Knative workloads._
+
+The [Knative Operator](https://knative.dev/docs/install/operator/knative-with-operators/) gives you a declarative API to describe your Knative Serving and Eventing installation. The `spec` field has several properties to define the desired behavior.
+
+### A convenient global configuration for high availability
+
+Take a look at the following installation manifest:
+
+```yaml
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  high-availability:
+    replicas: 3
+```
+
+This configures Knative Eventing in the `knative-eventing` namespace, and defines that all workloads, managed by the Operator, do require a replica set of `3` pods. Let's have a look:
+
+```bash
+kubectl get deployments.apps -n knative-eventing 
+NAME                    READY   UP-TO-DATE   AVAILABLE   AGE
+eventing-controller     3/3     3            3           58s
+eventing-webhook        3/3     3            3           57s
+imc-controller          3/3     3            3           53s
+imc-dispatcher          3/3     3            3           52s
+mt-broker-controller    3/3     3            3           51s
+mt-broker-filter        3/3     3            3           51s
+mt-broker-ingress       3/3     3            3           51s
+```
+
+For each workload we do see exactly three deployements. Now, take a detailed look at the above shell snippet. You will notice that for the `InMemoryChannel` we do have `6` deployments: `3` for each, the `controller` and the `dispatcher` data-plane. This is not always what you want, since the `InMemoryChannel` is more often used as a tool during development, while in production scenarios other worksloads, like the K[native Broker](https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/) or [Knative Channel](https://knative.dev/docs/eventing/configuration/kafka-channel-configuration/) implementations for Apache Kafka are being used. 
```

Comment:
```suggestion
For each workload we do see exactly three deployements. Now, take a detailed look at the above shell snippet. You will notice that for the `InMemoryChannel` we do have `6` deployments: `3` for each, the `controller` and the `dispatcher` data-plane. This is not always what you want, since the `InMemoryChannel` is more often used as a tool during development, while in production scenarios other worksloads, like the [Knative Broker](https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/) or [Knative Channel](https://knative.dev/docs/eventing/configuration/kafka-channel-configuration/) implementations for Apache Kafka are being used. 
```